pvacseq/lib/call_iedb.py

```diff
@@ -42,10 +43,14 @@ def main(args_input = sys.argv[1:]):
             response = requests.post('http://tools-api.iedb.org/tools_api/mhci/', data=data)
     if response.status_code != 200:
         sys.exit("Error posting request to IEDB.\n%s" % response.text)
-    args.output_file.write(response.text)
+
+    tmp_output_file = args.output_file + '.tmp'
+    tmp_output_filehandle = open(tmp_output_file, 'w')
+    tmp_output_filehandle.write(response.text)
+    tmp_output_filehandle.close()
+    os.rename(tmp_output_file, args.output_file)
```

Comment:
I think [os.replace](https://docs.python.org/3/library/os.html#os.replace) would be better here for cross-platform support (os.rename fails on windows if the destination exists, os.remove will overwrite the destination if it exists on any platform)